extensions/parquet/table/src/test/java/io/deephaven/parquet/table/ParquetTableReadWriteTest.java

```diff
@@ -87,6 +88,9 @@ public void setUp() {
         }
         // noinspection ResultOfMethodCallIgnored
         rootFile.mkdirs();
+
+        // Allow a small page size for testing
+        Configuration.getInstance().setProperty("Parquet.minTargetPageSize", "64");
```

Comment:
This isn't going to work reliably. If `ParquetInstructions` has been statically initialized, this will be ineffective. Why not just amend the tests below to set `MIN_TARGET_PAGE_SIZE` as the default, and try to write pages bigger than that?

Reply:
Will do!